src/Dfc.CourseDirectory.WebV2/Features/Courses/DeleteCourseRun.cshtml

```diff
@@ -0,0 +1,87 @@
+﻿@model Dfc.CourseDirectory.WebV2.Features.Courses.DeleteCourseRun.ViewModel
+@{
+    Layout = "_V2LayoutProviderContext";
+    ViewBag.Title = "Delete course";
+
+    var returnUrl = ViewData["ReturnUrl"] ?? "/";
+}
+
+@if (!ViewContext.ModelState.IsValid)
+{
+    <govuk-error-summary>
+        <govuk-error-summary-title>There is a problem</govuk-error-summary-title>
+        <govuk-error-summary-item asp-for="Confirm" />
+    </govuk-error-summary>
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <form asp-action="DeleteCourseRun" asp-route-courseId="@Model.CourseId" asp-route-courseRunId="@Model.CourseRunId" asp-route-returnUrl="@returnUrl">
+            <h1 class="govuk-heading-xl">Delete course</h1>
+
+            <p class="govuk-body">Deleting this course will remove it from Publish to the course directory and National Careers Service, Find a course.</p>
```

Comment:
Long lines make diffs hard to review in github. Sad but true. Suggest wrapping this file's long lines.